applications/smart-distancing/eval/README.md

```diff
@@ -0,0 +1,91 @@
+# Smart Social Distancing - Performance Metrics for Object Detection
```

Comment:
I found that there are two possible workflow in this repo:

1. To Evaluate the detection result which are ready to use (in the format you have explained in Readme)
2. To run tflite model using `edgetpu_eval.py` and test data to generate test result and then evaluate results

but I can't find these information in the Readme file. please provide more basic information in Readme about the workflow of provided scripts and dockers.

Reply:
Yes, other workflows will be provided such as test on coral-dev devices, jetson-nano, GPU, ...